configuration/log-configuration.yaml

```diff
@@ -181,3 +181,9 @@ TraceTxOutbound: True
 
 # Trace TxSubmission protocol messages.
 TraceTxSubmissionProtocol: True
+
+# Trace Handshake protocol messages.
```

Comment:
Sorry @karknu these also need to be in alphabetical order.

Reply:
The time sorting trace options would have been better spent  coming up with a better solution to having 15 near identical yaml files.

Reply:
Agreed, these yaml files will reduce once the benchmarking stuff is out of the repo.